docs/intro.txt

```diff
@@ -170,6 +170,7 @@ Every :py:class:`Backend` in pytket has its own ``default_compilation_pass`` met
      - self.rebase_pass [2]
      - `SynthesiseTket <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SynthesiseTket>`_
    * -
+     -
      - self.rebase_pass [2]
```

Comment:
I think when the table builds it shows the rebase pass applied after `RemoveRedundancies` (for optimisation level 1)

Shouldn't this be the other way round?

Reply:
Ah okay, I must've got the ordering wrong initially then.

Thanks for fixing.